flink-runtime-web/web-dashboard/app/scripts/index_hs.coffee

```diff
@@ -0,0 +1,193 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+angular.module('flinkApp', ['ui.router', 'angularMoment', 'dndLists'])
+
+# --------------------------------------
+
+.run ($rootScope) ->
+  $rootScope.sidebarVisible = false
+  $rootScope.showSidebar = ->
+    $rootScope.sidebarVisible = !$rootScope.sidebarVisible
+    $rootScope.sidebarClass = 'force-show'
+
+# --------------------------------------
+
+.value 'flinkConfig', {
+  jobServer: ''
+#  jobServer: 'http://localhost:8081/'
+  "refresh-interval": 10000
```

Comment:
The default UI refresh interval is larger than the history server refresh interval. I think both should be the same. I would go with 10s for both.